core/rio/benchmarks/src/test/java/org/eclipse/rdf4j/rio/benchmarks/RDFTestDataset.java

```diff
@@ -0,0 +1,213 @@
+/*******************************************************************************
+ * Copyright (c) 2021 Eclipse RDF4J contributors.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Distribution License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ *******************************************************************************/
+
+package org.eclipse.rdf4j.rio.benchmarks;
+
+import java.io.BufferedInputStream;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.UncheckedIOException;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.zip.GZIPInputStream;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipFile;
+
+import org.eclipse.rdf4j.common.io.IOUtil;
+
+/**
+ * This enum holds locations for RDF files from the web. It allows downloading these into a a temporary location in
+ * {@code {java.io.tmpdir}/rdf4j-benchmark-datasets/{filename} }. Please note
+ *
+ * @author Frens Jan Rumph
+ */
+public enum RDFTestDataset {
+
+	/**
+	 * <p>
+	 * A dump of the <a href="http://wifo5-03.informatik.uni-mannheim.de/bizer/berlinsparqlbenchmark/">Berlin SPARQL
+	 * Benchmark</a> made available by the <a href="https://project-hobbit.eu">HOBBIT project</a>.
+	 * </p>
+	 * <p>
+	 * 25 GB extracted.
+	 * </p>
+	 */
+	BSBM("bsbm.nt", "dataset.nt",
+			"https://hobbitdata.informatik.uni-leipzig.de/benchmarks-data/datasets-dumps/bsbm-dump.zip"),
+	/**
+	 * <p>
+	 * A dump of the <a href="http://owl.cs.manchester.ac.uk/publications/supporting-material/fishmark/">FishMark</a>
+	 * benchmark made available by the <a href="https://project-hobbit.eu">HOBBIT project</a>.
+	 * </p>
+	 * <p>
+	 * 1.8 GB extracted
+	 * </p>
+	 */
+	FISHMARK("fishmark.nt", "fishmark-size1.nt",
+			"https://hobbitdata.informatik.uni-leipzig.de/benchmarks-data/datasets-dumps/fishmark-dump.zip"),
```

Comment:
Thanks, let me know if there is anything I can do here.

Reply:
I went through the list you added and as far as I was able to determine most are covered by sufficiently permissive licenses (MIT, CC BY-SA, etc). Also, since the setup is such that we merely download on the fly for a test run then throw away (i.e. we don't actually have the dataset in our code base or distribute it ourselves), I think we're fine.

Reply:
Thanks!